lib/FieldType/XmlText/Converter/RichText.php

```diff
@@ -275,7 +275,10 @@ private function addXhtmlClassValue(DOMNode $node, $value)
     {
         $classAttributes = $node->attributes->getNamedItemNS('http://ez.no/xmlns/ezpublish/docbook/xhtml', 'class');
         if ($classAttributes == null) {
-            $node->setAttribute('ezxhtml:class', 'ez-embed-type-image');
+            $isLinked = $node->lastChild->nodeName === 'ezlink';
+            $class = 'ez-embed-type-image';
+            $class .= $isLinked ? ' is-linked' : '';
+            $node->setAttribute('ezxhtml:class', $class);
```

Comment:
Personally, I'd move `$isLinked` declaration before it is used.

My reasoning for this is that we're trying to establish ("building") what `$class` the attribute should be. That's the purpose of the code in question. Suffixing class with `"is-linked"` is a special case.

```suggestion
            $class = 'ez-embed-type-image';
            
            $isLinked = $node->lastChild->nodeName === 'ezlink';
            $class .= $isLinked ? ' is-linked' : '';

            $node->setAttribute('ezxhtml:class', $class);
```